public/app/diarization-timeline.js

```
@@ -0,0 +1,457 @@
+// Rendering
+// ─────────────────────────────────────────────
+
+function _render() {
+  const { canvas, ctx, segments, speakers, duration,
+          viewStart, viewDuration, currentTime, activeSpeaker } = Timeline;
+  if (!canvas || !ctx || duration === 0) return;
+
+  const W = canvas.width;
+  const H = canvas.height;
+  const speakerIds = Object.keys(speakers);
+  const trackCount  = speakerIds.length || 1;
```

ESLint

[error] 'trackCount' is assigned a value but never used.

(no-unused-vars)